scripts/test/updateAtlas.js

```diff
@@ -22,7 +22,7 @@ async function main() {
         throw new Error("Cannot find a tests/testProject. Did you run the script in the widget folder?");
     }
 
-    rm("-rf", "tests/testProject/theme", "tests/testProject/themesource");
+    rm("-rf", "tests/testProject/theme", "tests/testProject/themesource/atlas*");
```

Comment:
Leo added a logic to only override the files, so please remove this line from the branch 😄 

https://github.com/mendix/widgets-resources/blob/d4f978d3ac8bc6b8694cbf873211e77da3676b16/scripts/test/updateAtlas.js#L52

Reply:
Removing the Atlas themesource folders first ensures removed Atlas src files will not be present anymore in the test project Atlas themesource folders. So I think we should stick to your first suggestion :)